velox/docs/functions/presto/aggregate.rst

```diff
@@ -214,6 +214,12 @@ Bitwise Aggregate Functions
 
     Returns the bitwise OR of all input values in 2's complement representation.
 
+.. function:: bitwise_xor_agg(x) -> [same as input]
+
+    Returns the bitwise XOR of all input values in 2's complement representation.
+
+    Supported types are TINYINT, SMALLINT, INTEGER and BIGINT.
```

Comment:
I noticed recently that Presto only supports BIGINT. Is this the case?

Reply:
Yes, I commented this fact in another PR.